### ISSUE TITLE
Undefined variable $entryType breaking element integration in Craft 5

### ISSUE THREAD

**Issue by mike-moreau** [verbb/formie#1916]
### Describe the bug

Attempting to connect a form with an Elements integration in Craft 5 is resulting in the following error for me:
```
Failed to trigger integration: ["Element integration failed for submission “148”. Error: Undefined variable $entryType .../vendor\/verbb\/formie\/src\/integrations\/elements\/Entry.php:238"].
```

### Steps to reproduce

1. Create a form
2. Set up elements integration
3. Submit the form, the queue job should fail with this error

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

5.1.9

### Plugin version

3.0.0-beta.14

### Multi-site?

No

**Comment by mike-moreau** [comment]
I wonder if this line 238 should be `$entry` instead of `$entryType`?

```
$siteSettings = ArrayHelper::firstWhere($entry->section->getSiteSettings(), 'siteId', $entry->siteId);
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.14"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.15](https://github.com/verbb/formie/releases/tag/3.0.0-beta.15)
